src/mscorlib/src/Internal/Runtime/Augments/EnvironmentAugments.cs

```diff
@@ -17,5 +18,11 @@ public static class EnvironmentAugments
         public static bool HasShutdownStarted => Environment.HasShutdownStarted;
         public static string StackTrace => Environment.StackTrace;
         public static int TickCount => Environment.TickCount;
+        public static string GetEnvironmentVariable(string variable) => Environment.GetEnvironmentVariableCore(variable);
```

Comment:
It's factored out to ensure there is correct error handling. As we're not intending to use the Win32 version from core it gets weird any way we do it. I can either duplicate the error checking or refactor it into separate places. This seemed like the lesser of evils to me.